spec/views/records/edit_fields/_resource_type.html.erb_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'rails_helper'
```

Comment:
These aren't necessary since `.rspec` provides this by default.

Reply:
Im just trying to not break the way we have things now. all the other ones have it too. Im trying to balance going in and fixing everything and getting my work in